tests/system/common.py

```diff
@@ -676,11 +676,30 @@ def http_get_marathon_path(name, marathon_name='marathon'):
 # https://github.com/dcos/dcos-cli/pull/974
 def delete_marathon_path(name, marathon_name='marathon'):
     """Invokes HTTP DELETE for marathon url with name.
-       For example, name='v2/leader': http GET {dcos_url}/service/marathon/v2/leader
+       For example, name='v2/leader': http DELETE {dcos_url}/service/marathon/v2/leader
     """
     url = get_marathon_endpoint(name, marathon_name)
     return http.delete(url)
 
+def abdicate_marathon_leader(params = "", marathon_name='marathon'):
+    """
+    Abdicates current leader
+
+    Accounts for Marathon abdication delay of 0.5 seconds. Waits for just a little longer.
+
+    params arg should include a "?" prefix.
+    """
+    leader_endpoint = get_marathon_endpoint('/v2/leader', marathon_name)
+    result = http.delete(leader_endpoint + params)
+
+    down=False
+    while not down:
```

Comment:
We prefer `@retrying.retry` methods to the loops. See here for example:
https://github.com/mesosphere/marathon/pull/6294/files#diff-984c5562f7187dee9e6d3e161c7b52f8L116

Reply:
✅